### DISCUSSION TITLE
I don't understand the syntax of -motion-blur

### DISCUSSION THREAD

**Discussion by MrDowntempo** [ImageMagick/ImageMagick#6056] [Help]
### ImageMagick version

7.1.0-61-Q16-HDRI-x64

### Operating system, version and so on

Win10 x64

### Description

I am trying to motion blur some images in batch. I can get the motion blur working, but I need to change the angle. However, the documentation's shows some weird syntax. radius{xsigma}+angle. I know the radius I want is 10, and the angle is presumably 90 degrees (documentation is explicit about angle units) But anything I try like magick mogrify -motion-blur 10 +90 *.png
or 10+90 or 10{}+90 doesn't work. I'm not sure how mogrify needs the command formatted and the documentation unfortunately hasn't helped.

**Comment by fmw42** [comment]
Try 0xsigma+angle.  IM will compute the radius from the sigma as approx r=3sigma.

**Comment by snibgo** [comment]
`radius{xsigma}+angle` means we need three numbers, with `x` after the first and `+` after the second, with no spaces. `{}` means the `x` and the number for sigma are optional. So the following examples would be valid:
```
magick in.png -motion-blur 0x5+90 out.png
magick in.png -motion-blur 15x5+90 out.png
magick in.png -motion-blur 15+90 out.png
```

**Reply by MrDowntempo** [reply]
Ah ok, this makes more sense now. I didn't see the x as separate from sigma. What exactly is sigma describing?
To be clear. What I'm specifically trying to do is what I'd get from a 10px motion blue in GIMP or Photoshop

**Comment by fmw42** [comment]
It is a measure of the blur width. It is like standard-deviation in some sense. Look up Gaussian or Normal blur on Wikipedia. It describes the sigma of the normal curve, which describes a Gaussian blur as well. It marks a certain width of the bell shape curve where it has dropped a certain amount from the peak.
